docs/line.asciidoc

```diff
@@ -27,9 +27,27 @@ NOTE: In Elasticsearch releases 1.4.3 and later, this functionality requires you
 {ref}/modules-scripting.html[dynamic Groovy scripting].
 
 The availability of these options varies depending on the aggregation you choose.
-Select *view options* to change the following aspects of the table:
 
+Select the *Options* tab to change the following aspects of the chart:
+
+*Y-Axis Scale*:: You can select *linear*, *log*, or *square root* scales for the chart's Y axis.
+*Smooth Lines*:: Check this box to curve the line from point to point.
```

Comment:
Its worth pointing out to the user some of the downsides with curved lines: http://vizwiz.blogspot.com/2011/12/when-you-use-smoothed-line-chart-your.html